scripts/write_gcr_to_parquet.py

```diff
@@ -96,8 +96,8 @@ def write_dataframe_to_files(
     # Append iff the file already exists
     parquet_append = append and os.path.exists(output_filename)
     df.to_parquet(output_filename,
-                  append=parquet_append,
-                  file_scheme=parquet_scheme,
+                  #append=parquet_append,
+                  #file_scheme=parquet_scheme,
```

Comment:
Why are these lines connected to this PR?

Reply:
I think this change should be a separate, trivial PR.  Then we we want to revert it with an updated `pyarrow` it will be easy to identify the change.